docs-public/snippets.js

```diff
@@ -1180,7 +1194,7 @@ window.openInPlunker = function(listing) {
         }, listing, files);
     })
     .then(function(files) {
-        var form = buildExampleEditorForm(exampleTemplate);
+        var form = buildExampleEditorForm(exampleTemplate, files);
```

Comment:
The `files` parameter is used only for building the dependencies that need to be posted with the form. This introduces more responsibilities to the `buildExampleEditorForm` function ("Build the form _and_ generate the dependency list"). A better way to do this is to pass the dependency list directly, and have a function that maps files to dependencies. This is also easier to unit test (_wink, wink_) :smiley: